docs/Developers/Linter_Bears_Advanced.rst

```diff
@@ -95,24 +95,70 @@ several functions accessible with the naming scheme
 ``process_output_<output-format>``.
 
 - ``process_output_regex``: Extracts results using a regex.
+
+  ::
+
+      @linter(executable='my_tool',
+              use_stderr=True)
+      class MyBear:
+          # Assuming the tool puts some issue messages into stderr.
+          def process_output(self, output, filename, file):
+              # output is a string, as we activated just ``use_stderr``
+              map = {'info': RESULT_SEVERITY.INFO,
+                     'warn': RESULT_SEVERITY.NORMAL,
+                     'error': RESULT_SEVERITY.MAJOR}
```

Comment:
isn't this default?

Reply:
Yes, it is. 
These tutorials are intended for new developers, so I thought explicitly specifying that the ``process_output_regex`` method can intake a ``map`` parameter will defer the need to go through the API specs.